src/main/java/forestry/farming/multiblock/FarmController.java

```diff
@@ -558,7 +558,7 @@ private boolean cullCrop(ICrop crop, IFarmLogic provider) {
 			}
 		}
 
-		final int fertilizerConsumption = provider.getFertilizerConsumption();
+		final int fertilizerConsumption = (int)(provider.getFertilizerConsumption() * Config.fertilizerModifier);

```

Comment:
I think in some cases it may be important to use `Math.round(` instead of casting to `(int)` for these. Better to use it and be more correct.

Reply:
https://github.com/ForestryMC/ForestryMC/pull/1948/commits/05d1eb87ce1132fb82e2c60d10324697ab2d44da